internal/pkg/skuba/util/cert.go

```diff
@@ -94,6 +94,7 @@ func CreateOrUpdateCertToSecret(
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      secretName,
 			Namespace: metav1.NamespaceSystem,
+			Labels:    map[string]string{"caasp.suse.com/skuba-addon": "true"},
```

Comment:
`CreateOrUpdateCertToSecret` is a general function basically not just for addon usage in the future. I would suggest to have a call back or argument to update the labels of the secret.